packages/jest-runtime/src/__tests__/Runtime-sourceMaps.test.js

```diff
@@ -4,7 +4,6 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
```

Comment:
The blank line under license can probably go too, yeah?

Reply:
A few `.ts` files that I looked at for example have one:

```js
 * LICENSE file in the root directory of this source tree.
 *
 */
```

Reply:
Yeah it's totally inconsistent across the code base. I've learned to ignore it ^^

Reply:
lazy removal of `@flow` pragmas

Reply:
Yes, that is a tactful way to say that I was consistently lazy, bro ❤️

Reply:
😀